README.md

```diff
@@ -123,49 +148,29 @@ using System.Collections.Generic;
 
 public class MyCoolBehaviorScript : MonoBehaviour {
     void Start () {
-        Branch.initSession(CallbackWithParams);
```

Comment:
@aaustin hey aaustin, im trying to implement the new v3.0 inside our products atm. Can you explain why its not recommended to use universal object callbacks (if i understood you correctly :D)?